.github/workflows/linear-story-update.yml

```
@@ -0,0 +1,200 @@
+            echo "story_id=$STORY_ID" >> $GITHUB_OUTPUT
+            echo "Found Linear Story ID: $STORY_ID"
+          else
+            echo "No Linear Story ID found"
+          fi
+
+      - name: Update Linear Story Progress
+        if: steps.extract-story.outputs.story_id != ''
+        run: |
+          set -e
+          STORY_ID="${{ steps.extract-story.outputs.story_id }}"
+          
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)